src/overdub.jl

```diff
@@ -144,7 +144,10 @@ function overdub_pass!(reflection::Reflection,
     #=== execute user-provided pass (is a no-op by default) ===#
 
     if !iskwfunc
-        code_info = passtype(context_type)(context_type, reflection)
+        # passtype is defined in the wrong world-age so we need to run it in the newest
+        # ideally we would add a edge from that function to the generated thunk, but
+        # alas that would require us, to trace all dynamic function calls. Here be dragons.
+        code_info = Core._apply_pure(passtype(context_type), (context_type, reflection))
```

Comment:
Question, why isn't `invokelatest` enough here? Why does it need to be `apply_pure`?

All we really want is to run it in the latest world age, right?

Reply:
Oh. Right. Duh. It's because this is being called from inside a generated function, where `invokelatest` has no effect. That's the whole reason we wrote this code once we realized what `_apply_pure` could do... NVM sorry. :)